tutorial.html

```diff
@@ -14,13 +14,70 @@
         <script src="js/tutorial.js"></script>
 
         <link rel="shortcut icon" href="favicon_grey.ico" type="image/x-icon">
-        <script>
-            $(window).on('load', function() {
-                $("#cover").hide();
-            });
-        </script>
     </head>
     <body>
+        <div id="absolute_logo">
+            <img id="logo_hebbia" opacity="80%" src="img/hebbia_logo.png">
+        </div> 
+
+
+        <div class="signup" id="signup_p1">
+            <div class="signup_content signup_header">
+                Hebbia Is Installed
+            </div>
+            <div class="signup_content signup_subheader">
+                Let's get started!
+                <!--Let's start by creating your account.-->
+            </div>
+
+            <form class="signup_content" name="enter_email" id="enter_email" action="#">
+                <input id="email" name="email" type="email" placeholder="Your Email" required>
+                <button class="glow-on-hover" type="submit">
+                    <span>
+                        Sign up
+                    </span>
+                </button>
+            </form>
+            <div class="signup_content signup_text_small">
+                By signing up, you agree to Hebbia's <a href="https://hebbia.ai/terms">Terms and Conditions</a> and <a href="https://hebbia.ai/privacy-policy">Privacy Policy</a>. <!--You also agree to receive product-related marketing emails from Hebbia, which you can unsubscribe from at any time.-->
+            </div>
+            <div class="signup_content signup_text_small signup_later">
+                Sign up later
+            </div>
+        </div>
+
+        <div class="signup" id="signup_p2" style="display: none;">
+            <div class="signup_content signup_header">
+                Customize Your Experience
```

Comment:
Idk about this rewording. This would imply that the user is customizing their own experience by entering this information, which seems unnatural and more appropriate for something like our Options page. I could see how the user is technically customizing their experience indirectly, but I would personally be quite thrown off by this wording.